pdoc/doc.py

```diff
@@ -151,6 +151,20 @@ def source_file(self) -> Optional[Path]:
         except TypeError:
             return None
 
+    @cached_property
+    def source_lines(self) -> Optional[tuple[int, int]]:
+        """
+        Return a `(start, end)` line number tuple for this object.
+
+        If no source file can be found, `None` is returned.
+        """
+        try:
+            obj = inspect.unwrap(self.obj)  # type: ignore
```

Comment:
`inspect.unwrap()` is not needed here, because it's called by `inspect.getsourcelines(obj)`

Reply:
Good catch, thanks!